spectractor/parameters.py

```diff
@@ -111,8 +96,8 @@ def __getattr__(name):
 LAMBDA_MIN = 300  # minimum wavelength for spectrum extraction (in nm)
 LAMBDA_MAX = 1100  # maximum wavelength for spectrum extraction (in nm)
 LAMBDA_STEP = 0.2  # step size for the wavelength array (in nm)
-LAMBDAS = np.arange(LAMBDA_MIN, LAMBDA_MAX, LAMBDA_STEP)
 SPEC_ORDER = 1  # spectrum order to extract
+LAMBDAS = np.arange(LAMBDA_MIN, LAMBDA_MAX, 1)
```

Comment:
1. yes but unfortunately some doctests uses parameters.LAMBDAS so I prefer to keep this array in parameters.py
2. but you're right I should use LAMBDA_STEP to avoid confusion
3. I add the use of LAMBDA_STEP in `update_derived_parameters()` , it is a mistake